lib/ruboty/slack_socket_mode/interactive_message.rb

```diff
@@ -46,6 +46,37 @@ def delete
 
         robot.delete_interactive(response_url)
       end
+
+      def update(text: nil, block: nil)
```

Comment:
[imo] reply で Block Kit のUIを指定する際は `blocks` としているので、揃えたいです！

```suggestion
      def update(text: nil, blocks: nil)
```

Reply:
その通り過ぎてblocksにしました。